File: Lab Practice 1/BalanceAmount.c
#include <stdio.h>
int main() {

    double principalAmount, annualInterestRate, balance;
    printf("Enter the principal amount: ");
    scanf("%lf", &principalAmount);

    printf("Enter the annual interest rate (in percentage): ");
    scanf("%lf", &annualInterestRate);
    annualInterestRate /= 100;

    for (int year = 1; year <= 5; year++) {
        balance = principalAmount * (1 + annualInterestRate);

        printf("Balance at the end of year %d: %.2f\n", year, balance);

        principalAmount = balance;
    }
    return 0;
}

File: Lab Practice-1/BalanceAmount.java
import java.util.Scanner; 
 
public class investment { 
    public static void main(String[] args) { 

        Scanner scanner = new Scanner(System.in); 

        System.out.print("Enter the initial amount (Principal amount): "); 
        double principalAmount = scanner.nextDouble(); 
 
        System.out.print("Enter the interest rate (in percentage): "); 
        double interestRate = scanner.nextDouble(); 
 
        interestRate /= 100; 
 
        System.out.println("\nYear\tBalance"); 
 
        for (int year = 1; year <= 5; year++) { 
            double balance = principalAmount * Math.pow(1 + interestRate, year); 

            System.out.printf("%d\t%.2f%n", year, balance); 
        } 
        scanner.close(); 
    } 
}
